Google.Api.Generator.Tests/Invoker.cs

```diff
@@ -24,6 +24,10 @@ namespace Google.Api.Generator.Tests
 {
     internal static class Invoker
     {
+        const string ProtocFileStr = "ProtocFile";
+        const string RuntimeStr = "Runtime";
+        const string PluginFileStr = "PluginFile";
```

Comment:
We always specify visibility modifiers, and we rarely include an indication of the type on the name of the variable. You can use `Key` as a suffix instead

```suggestion
        private const string ProtocFileKey = "ProtocFile";
        private const string RuntimeKey = "Runtime";
        private const string PluginFileKey = "PluginFile";
```

Reply:
Done!